src/pretix/base/settings.py

```diff
@@ -464,7 +480,48 @@ def unserialize(cls, s):
             label=_('Only end payment terms on weekdays'),
             help_text=_("If this is activated and the payment term of any order ends on a Saturday or Sunday, it will be "
                         "moved to the next Monday instead. This is required in some countries by civil law. This will "
-                        "not effect the last date of payments configured above."),
+                        "not effect the last date of payments or a term by minutes configured above."),
+            widget=forms.CheckboxInput(
+                attrs={
+                    'data-display-dependency': '#id_payment_term_mode_0',
+                    'data-required-if': '#id_payment_term_mode_0'
+                },
+            ),
+        )
+    },
+    'payment_term_minutes': {
+        'default': '30',
+        'type': int,
+        'form_class': forms.IntegerField,
+        'serializer_class': serializers.IntegerField,
+        'form_kwargs': dict(
+            label=_('Payment term in minutes'),
+            help_text=_("The number of minutes after placing an order the user has to pay to preserve their reservation. Only use this for real-time "
+                        "payment methods."),
```

Comment:
Maybe: "Only use this if you exclusively offer real-time payment methods."? Otherwise sounds as if you could do it per-payment-method.